menpo/landmark/labels/human/face.py

```diff
@@ -276,6 +276,70 @@ def face_ibug_68_to_face_ibug_49(pcloud):
     return new_pcloud, mapping
 
 
+@labeller_func(group_label='face_ibug_49')
+def face_ibug_49_to_face_ibug_49(pcloud):
```

Comment:
just document this addition in the PR description if you want to add it here, so it's discoverable when we come to write up the changelog